pom.xml

```diff
@@ -1724,14 +1724,26 @@
                         <groupId>org.apache.pinot</groupId>
```

Comment:
can you check if you can remove the shaded classifier on line 1721 and keep those `pinot-*-jdk8` dependencies?

Reply:
There are some dependency failure by enforcer. Will try more and ping you later
(Some sample error:
```
Require upper bound dependencies error for org.checkerframework:checker-qual:2.5.2 paths to dependency are:
+-com.facebook.presto:presto-pinot-toolkit:0.273-SNAPSHOT
  +-com.google.guava:guava:26.0-jre
    +-org.checkerframework:checker-qual:2.5.2
and
+-com.facebook.presto:presto-pinot-toolkit:0.273-SNAPSHOT
  +-org.apache.pinot:presto-pinot-driver:0.10.0
    +-org.apache.pinot:pinot-common-jdk8:0.10.0
      +-org.apache.calcite:calcite-core:1.29.0
        +-org.checkerframework:checker-qual:3.10.0
```